extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -415,8 +442,9 @@ export class AmpConsent extends AMP.BaseElement {
    * @param {string} action
    * @param {string=} consentString
    * @param {!ConsentMetadataDef=} opt_consentMetadata
+   * @param {!../../../src/service/action-impl.ActionInvocation=} opt_invocation
    */
-  handleAction_(action, consentString, opt_consentMetadata) {
+  handleAction_(action, consentString, opt_consentMetadata, opt_invocation) {
```

Comment:
How about we change this method to accept an object with the current params as keys? Then you don't have to pass `undefined` like in the cases above? May be a departure from how we currently pass params in AMP. But since this is a private method this might be a good place to start?

Reply:
see comment below